Automattic/pocket-casts-android
# Support edge-to-edge for the filters section

## Description

This change focuses on all the screens in the filters tab. Please ignore the other tabs for now.

There will be multiple pull requests required to support Android 15 edge-to-edge but I'm going to break them down into smaller PRs. I will merge them all into a `task/android-15` branch, and after all the work is completed, raise a PR to merge into `main`.

## Testing Instructions

Please try tapping around in the filters tab to find issues with the edge-to-edge changes. Areas I have discovered issues with are:
- Older versions of Android versions
- Themes
- Content under the system bars 
- End of pages / lists under the bottom navigation or mini player

Here are some test steps I have followed in case you want to try them too:

1. Open the filters tab
2. ✅ Verify the next pages aren't under the status bar and you can get to the bottom of the page, even with the mini player open.
3. Tap on a podcast
4. Tap on the podcast settings cog
5. Tap on Auto archive and back
7. Tap on Playback effects and back
8. Tap on Filters and go back to the podcasts list
9. Tap on the search podcasts icon, search for a podcast, and come back to the list
10. Tap on the three dots, share podcasts, and go through the screens to create a list

## Screenshots

| Before | After |
| --- | --- |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |

## Checklist
 
#### I have tested any UI changes...
